tests/snippets/mcfunction/commenting.txt

```diff
@@ -0,0 +1,293 @@
+---input---
+#> Get: #rx.playerdb:api/v2/get
+#>
+#> @input
+#>  $in.uid rx.io
+#>
+#> @output
+#>  rx.playerdb:io player
+#
+# Selects data inside the database and copies to rx.playerdb:io player
+# See #api/v2/select for more info..
+# 
+#* Note: something, something, this is important..
+
+# Normal Comment
+
+# This **shouldn't** be a comment..
+scoreboard players operation @s obj = #fakeplayer obj
+
+#> single line block comment
+tellraw @a "This string # has # hashtags o_O"
+
+---tokens---
+'#>'          Comment.Multiline
+' '           Text.Whitespace
+'Get:'        Literal.String.Doc
+' '           Text.Whitespace
+'#rx.playerdb:api/v2/get' Name.Function
+'\n'          Text
+
+'#>'          Comment.Multiline
+'\n'          Text
+
+'#>'          Comment.Multiline
+' '           Text.Whitespace
+'@input'      Name.Decorator
+'\n'          Text
+
+'#>'          Comment.Multiline
+'  '          Text.Whitespace
+'$in.uid'     Name.Variable.Magic
+' '           Text.Whitespace
+'rx.io'       Literal.String.Doc
+'\n'          Text
+
+'#>'          Comment.Multiline
+'\n'          Text
+
+'#>'          Comment.Multiline
+' '           Text.Whitespace
+'@output'     Name.Decorator
+'\n'          Text
+
+'#>'          Comment.Multiline
+'  '          Text.Whitespace
+'rx.playerdb:io' Name.Function
+' '           Text.Whitespace
+'player'      Literal.String.Doc
+'\n'          Text
+
+'#'           Comment.Multiline
+'\n'          Text
+
+'#'           Comment.Multiline
+' '           Text.Whitespace
+'S'           Comment.Multiline
```

Comment:
This looks like a rule could use some repeats.

Reply:
I'm not sure what this means, should the tokens for commenting be larger (most text being coupled under a single token versus whitespace separating the words)?